clusterdock/topologies/cdh/actions.py

```diff
@@ -18,11 +18,12 @@
 from argparse import Namespace
 from collections import OrderedDict
 from datetime import datetime
+import io
```

Comment:
Please change to follow existing convention of putting all `import ...` imports before any `from ... import ...` lines. Same comment below, too.